mods/default/nodes.lua

```diff
@@ -1613,6 +1617,96 @@ minetest.register_node("default:bush_sapling", {
 	end,
 })
 
+minetest.register_node("default:huckleberry_bush_stem", {
+	description = "Huckleberry Bush Stem",
+	drawtype = "plantlike",
+	visual_scale = 1.41,
+	tiles = {"default_huckleberry_bush_stem.png"},
+	inventory_image = "default_huckleberry_bush_stem.png",
+	wield_image = "default_huckleberry_bush_stem.png",
+	paramtype = "light",
+	sunlight_propagates = true,
+	groups = {choppy = 2, oddly_breakable_by_hand = 1, flammable = 2},
+	sounds = default.node_sound_wood_defaults(),
+	selection_box = {
+		type = "fixed",
+		fixed = {-7 / 16, -0.5, -7 / 16, 7 / 16, 0.5, 7 / 16},
+	},
+})
+
+minetest.register_node("default:huckleberry_bush_leaves", {
```

Comment:
This needs to be named and described as having berries.
`"default:huckleberry_bush_leaves_with_berries"`?
Then we'll need a leaf node without berries as the nodetimer marker for berry regrowth.